quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportAtStartupDistTest.java

```diff
@@ -60,6 +62,7 @@ void testIgnoreFileWithUnsupportedExtension(LaunchResult result) {
     }
 
     @Test
+    @EnabledOnOs(value = { OS.LINUX, OS.MAC }, disabledReason = "different shell escaping behaviour on Windows.")
```

Comment:
If you try running it on your IDE (which should ignore the fact the test is targeted for Windows) you'll see that the message is different than what is expected. Piccoli does not validate arity on Windows for some reason. We can do a follow-up.

But if you are talking about the disabled reason, I agree it is misleading because I'm not sure if it is related to escaping.

Reply:
works for me, created https://github.com/keycloak/keycloak/issues/13501